pkg/features/kube_features.go

```diff
@@ -757,9 +757,10 @@ const (
 	// Enables topology aware hints for EndpointSlices
 	TopologyAwareHints featuregate.Feature = "TopologyAwareHints"
 
-	// owner: @lmdaly
```

Comment:
Also need to update types.go - remove wording `requires the TopologyManager feature gate to be enabled`

Reply:
Sure, will do now. Thanks!